core/common/src/main/java/alluxio/underfs/UnderFileSystemConfiguration.java

```diff
@@ -26,7 +29,8 @@
 public final class UnderFileSystemConfiguration {
```

Comment:
update class-level javadoc

Reply:
should this just be a subclass of `InstancedConfiguration`, with additional `readonly` `shared` and `ufsConf` fields?